docs/platform/release-notes/tizen-6-0-m2.md

```diff
@@ -0,0 +1,407 @@
+  - Array.prototype.sort has been updated due to the specification change.
+  - Function.prototype.toString has been updated due to the specification change.
+- Performance and Memory Optimization
+  - JS Engine
+    - Debugger core features have been updated.
+    - Lazy AtomicString initialization for memory/performance has been optimized.
+    - Vector extension logic has been improved.
+
+#### Fixes
+
+- Bug fixes
+  - Layout bugs have been fixed.
+  - Scroll bugs have been fixed.
+  - Font Rendering bugs have been fixed.
+  - Memory overflow bugs have been fixed.
+  - Crash issues have been fixed.
+
+
+### Tizen .NET
+
+#### New and changed features
+
+- NET runtime and tools
+  - .NETCore has been upgraded from 3.1.0 to 3.1.3.
+- TizenFx
+  - Tizen.NUI
+    - VertexBuffer class has been added.
+    - Pagination class has been added.
+    - Screen connection methods have been added to CustomView class.
+    - SlidingStarted event has been added to Slider class.
+    - ThumbColor property has been added to Slider class.
+    - FlexLayout and GridLayout have been added.
+    - ScrollPosition and ScrollCurrentPosition have been added to ScrollableBase class.
+    - Notification class has been added.
+    - FlexContainer class has been deprecated.
+    - IndicatorColor and SelectedIndicatorColor have been added to Pagination class.
+    - Quickpanel has been added to WindowSystem.
+  - Tizen.Network.Connection
+    - Required privilege level has been changed.
+  - Tizen.MachineLearning
+    - Custom filter class has been added.
+  - Tizen.Multimedia
+    - Setter/Getter for volume recording has been added to AudioIO.
+    - Result code and playback state have been added to MediaController.
+    - New RequestCommand for async has been added to MediaController.
+    - New barcode types have been added to Vision.
+  - Tizen.Applications
+    - Synchronous connection request and private file sharing have been added to RPCPort.
+  - Tizen.Network
+    - New Avrcp Control has been added to Bluetooth.
+
+
+### Toolchain
+
+#### New and changed features
+
+- Toolchain configure dumper support has been developed.
+  - "{package}-configure-dump.rpm" in binutils, glibc, and gcc have been provided with “dump_configure“ macro.
+- Build with armv8-a machine option has been added.
+- ASan has been enabled.
+  - Automated build, test with JIRA reporting process have been established.
+  - Libasansi feature for .NET ASan has been back-ported from Tizen 5.5.
+
+#### Fixes
+
+- Patches for CVE-2020-10029 have been back-ported in glibc from upstream.
+
+
+### Machine Learning
+
+#### New and changed features
+
+- ML/Training API (new)
+  - The ability to update and train deep neural networks on Tizen devices has been added.
+  - The ML/Training API allows application developers to describe their own neural network models and train the models or to re-train pre-trained models in run-time.
```

Comment:
Do we have to change this ML/Training API too or is it ok in context?

Reply:
@jijoongmoon yes,l

 Machine Learning Training API (new)

At both places

Reply:
- Machine Learning Training API (new)
  - The ability to update and train deep neural networks on Tizen devices has been added.
  - The Machine Learning Training API allows application developers to describe their own Neural Network models and train the models or to re-train pre-trained models in run-time.